data/private_key8.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
+MIIEvgIBADANBgkqhkiG9w0BAQEFAASCBKgwggSkAgEAAoIBAQDklADy04Tp/k0m
+YB6VYptm90CkRvIKFqKiWVqNFxfmOA1kp2lqIiurnF0tOnFAO3MKN9eHBtsVJtBF
+us82EHgtFz3drkQktrHkqBzpVKqfLYd3oGFI1NZO8waBRyiFixhN3isf61LDWnSd
+2CYzExwedbJSckoBkfMzHYLVG9MlYL8zjrBKHysm63PNQPMawx7DF9e3AuglNbNu
+UMiiJdbh3I6XDINumzrHW+JJRm5Mi+u3jwIEoJfKZPJ60yyb8vmeedooGOgQDDAU
+SMgI6SNvqq5BhYqbzZfAu4/p3PMm2AV2UPuv1XIpjETahEoytW4/3y9Elz7ux2dF
+JxUQc7XdAgMBAAECggEBALd08q3yhqMnr9Q1IVbUpdWmbQNszaa7NB0JilvjRRNJ
+ubYnh7vfYR2DimVamrrq/BIfb6S4I+5+EHtmzBigb1QBa24ZCChPWcgkBdBgW/EW
+IbwpxOsnqDoFZBsueZ91XDRjNEmSHgMc/qwRZ9dmZxMrc8woXq2T7pYAAZn5kyZT
+UzblpQiacvqD2mMDe7QLUcTOxGRCzNNpIW1u6QNPrksvZS7FmvVBhmtWxb3JuHZE
+KM1X5WLlkzr+alfXGixjTZxDqmmeHCNdYJKLzPvWrh7yqLsfwZAiOc7hLGFFoRxL
+u6R2u6MOS2kNrssh1hSOFCOheE8z6ZpBoIlqLRRLMiECgYEA/+j2NMML7C3I2rST
+rBAm0D4IHXRRFUWdYVYzKz7lZBDrUQmDZie/NRcOMuadZ9ivfoamUP1Qwljwwr6K
+YpKgbU1dzG4XgPq+ES96xLS06G11qilVLl6ilhFGmdKQ6NYKcrnsIuony4S82cYi
+XRbW/e3kXYsuCpS1rY/uXOSNXe8CgYEA5KiU16hAKPNW5z7EGZE3Ae7skCgbD4sd
+fribCHzyGk1aIWlZ4JoA6kEoG3KgWeQgzTLGQmZMumFdk/auKsYX5tf5uBT+sChg
+SMnhDf+8QtiezhAgfaVhJ/4NiHnfIGHzL181xf8jNEs28rn4t4qooG+fB4eXkpWV
+77JJx8rBNPMCgYEAs5Mh0XxczWqDWxUD2uWqls4wm5qXmzmyai820M3d4g4NUlw+
+nFyfy/1VtVomckyXOU37PzHCjd4jdESIjwsdEBYOD/ng/Fz3KtAKciXsHH1S6AkB
+APzfUjblN+OOVBR+WGsmzIlLS4DllFt4maIBU0avzQRxVnNr2VL6jKzElnkCgYAT
+Kya/JXWreSrR2+sokYng+ZyPQS1zkh/43ZTxkzfgiJeDsYc9Uuu8t5WbJ3LV1lAB
+o74HFvhbr4neu5FCOLUMPxLKIchwEBJyR6o8ODIaEHrPn355ygFpDt0nkb/owQ1X
+7U1eudLDsYJ7uqdlC5SzTmgwg2drfGrDW1jgDNlBjQKBgA0fWwTqpVsGYqGFTq3x
+ooumnSXsaCZqX3e1Qc3gsW6Xx6cCviyLeBlM/NiaWutdUtAOQhhPzx7yAEc4hlGb
+S+pAN9UBkvDvDPsffdojn+fkDHpHsDmVBei5LTPGTAQDZygntssGKd9aSHnYQhAC
+3BY9Erc+gqYmgoRCMY/118y/
+-----END PRIVATE KEY-----
```

Comment:
I've had some issues with GH bots flagging this. Not sure if this will become an issue, "workaround"s i've used in the past is either to generate the key on test, or to base64 encode it (yea.. I know...)